server/scripts/createAITAUser.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../models/index");
+const User = db.User;
+
+module.exports = async () => {
+  await User.create({
```

Comment:
I suggest you add the same things I am doing in the `importChallenges`. 

First, search for if the `AITA` user exists. If it already exists, do nothing. Else make one. 

Also, the `id` field is irrelevant. We are using `username` as primary key.